tests/e2e/adapters/MockSource.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Utopia\Tests;
+
+use Utopia\Transfer\Source;
+use Utopia\Transfer\Transfer;
+
+class MockSource extends Source
```

Comment:
```suggestion
class Mock extends Source
```

When a class extends a base class, it need not have the name of the base class in it. Remember to update the filename as well